compiler/tests/asm.rs

```diff
@@ -8,13 +8,23 @@ fn verify_asm<T: FieldElement>(file_name: &str, inputs: Vec<T>) {
     verify_asm_string(file_name, &contents, inputs)
 }
 
-fn halo2_proof(file_name: &str, inputs: Vec<Bn254Field>) {
+#[cfg(feature = "halo2")]
+fn get_prover() -> Option<Backend> {
+    Some(Backend::Halo2)
+}
+
+#[cfg(not(feature = "halo2"))]
+fn get_prover() -> Option<Backend> {
```

Comment:
```suggestion
fn prover() -> Option<Backend> {
```
Personal preference against `get_`